src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+import duke.command.Command;
+import duke.task.TaskList;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class Duke {
+
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+    private boolean isExit;
+
+    /**
+     * Constructor for a Duke instance.
+     * @param filePath file path for file to save/load save data.
```

Comment:
Please add an Empty line between description and parameter section for all JavaDocs. 🥲